GeoLib/DuplicateGeometry.cpp

```diff
@@ -23,7 +23,7 @@ namespace GeoLib
 {
 DuplicateGeometry::DuplicateGeometry(GeoLib::GEOObjects& geo_objects,
                                      std::string const& input_name,
-                                     std::string output_name)
```

Comment:
creation scenario: (creation with two moves) vs. (creation, pass-by-ref, and copy) => one copy less for value.
existing var scenario: (copy to ctor arg, move) vs (pass-by-ref, copy) => equal nr. of copies.